cli-feed-v4.json

```
@@ -24716,6 +24726,107 @@
           "default": "false"
         }
       ]
+    },
+    "4.77.0-inprocess": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimes": {
+        "dotnet": {
+          "net8": {
+            "displayInfo": {
+              "displayName": ".NET 8.0",
+              "hidden": false,
+              "displayVersion": "v4",
+              "targetFramework": ".NET",
+              "description": "LTS",
+              "endOfLifeDate": "2026-11-12T00:00:00Z"
```

this should be November 10, 2026 per https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core
```suggestion
              "endOfLifeDate": "2026-11-10T00:00:00Z"
```